src/v2/layouts/HomeScreen/ExistingRegistrationTable.jsx

```diff
@@ -0,0 +1,331 @@
+import DataGrid from '@pepcus/react-simple-data-grid';
+import Row from 'pepcus-core/lib/Row';
+import styled from 'styled-components';
+import PropTypes from 'prop-types';
+import Col from 'pepcus-core/lib/Col';
+import Button from 'pepcus-core/lib/Button';
+import { useState } from 'react';
+import Message from 'components/common/Message';
+import CustomLoader from 'components/common/CustomLoader';
+import isEqual from 'lodash/isEqual';
+import { updateMembersOptInApi } from './updateMembersOptInApi';
+import { existingRegistrationMetaData } from './existingRegistrationMetaData';
+import { AdminSecretKey } from '../../components/AdminSecretkey/secretKey';
+
+const ExistingRegistrationTable = ({ members, strings, onClickBackBtn }) => {
+	const {
+		MEMBER_DETAILS,
+		BACK,
+		SUBMIT,
+		OPT_IN_OR_OPT_OUT_SUCCESS_MESSAGE,
+		OPT_IN_OR_OPT_OUT_FAILED_MESSAGE,
+		NO_ALTERNATE_HINDI,
+		YES_HINDI,
+	} = strings;
+	const [isLoading, setIsLoading] = useState(false);
+	const [gridData, setGridData] = useState(members);
+	const [registrationSuccess, setRegistrationSuccess] = useState(null);
+	const [registrationError, setRegistrationError] = useState(null);
+	const [isSubmitDisable, setSubmitDisable] = useState(true);
+	const [initialGridData, setInitialGridData] = useState(members || []);
+	const { secretKey } = AdminSecretKey;
+
+	const getUpdatedMetaData = () => {
```

Comment:
If we define the purpose in the method name that would be great.
```suggestion
	const replaceCustomComponentInMeta = () => {
```